Resources/views/Macros/responsiveImg.html.twig

```diff
@@ -23,9 +24,7 @@
                 data-src="{{ thumbor(image.url, options.placeholder_filter) }}"
             {% endif %}
             class="{% if options.lazyload %}lazyload {% endif %}{{ options.class }}"
-            {% if image.alt %}
-                title="{{ image.alt }}"
-            {% endif %}
+            alt="{{ options.alt is not empty ? options.alt : image.alt }}"
```

Comment:
Ist es nicht etwas sehr roundabout, hier im Zweifel auf `image.alt` zu gehen, was in Zeile 6 via `{% set image = {'alt': '', 'title': ''}|merge(image) %}` ebenfalls schon ein Leerstring geworden sein könnte, statt das irgendwie eindeutiger in einem Statement zu machen? 🤨

Reply:
Verstehe ich nicht. "image.alt" ist ein Leerstring, wenn in image keine alt enthalten ist.